test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorFactoryTest.cs

```diff
@@ -1883,6 +1856,35 @@ public void ValidateTagHelperAttributeDescriptor_WithValidPrefix_ReturnsTrue(str
             return data;
         }
 
+        private static TagHelperDescriptor CreateTagHelperDescriptor(
```

Comment:
Right. Now things are less convenient.

In addition, why bother wrapping a single constructor call with a pseudo-helper? This method and the next one don't have a reason for being. Certainly aren't necessary with the existing `internal` constructors.